README.md

```diff
@@ -140,6 +141,22 @@ In Trace Analytics you can visualize the following tables.
 
 In Insights you can see the summary table for Insights. Clicking the InsightId will take you to AWS console.
 
+### Service map
+
+[TODO: image of the map in grafana]
+
+Service map shows the same data as a service map inside X-ray console. It needs Grafana v7.4+ to use it's Node graph visualization otherwise it will show the data in a simple table.
```

Comment:
```suggestion
Service map shows the same data as a service map inside X-ray console.

To display the service map:

- Use the [Node graph panel](LINK) visualization in Grafana 7.4 plus.
- Show the data in a simple table. This is the only option if the Node graph panel is unavailable.
```

Reply:
Not sure if this should mention the Explore? In explore in v7.4 you will automatically get the Node graph you don't need to select it.

Reply:
Do you mean that if you click a node that it opens the trace in Explore or that the service map visualization is available in Explore?

The former, definitely. I don't know if the latter is true.

Reply:
So both are true. But I guess here we are talking more about how and where you can show the service map and what are the prerequisites so I wonder if there should be  

- Use the [Node graph panel](LINK) visualization in Grafana 7.4 plus.
- Use Explore in Grafana 7.4 plus.
- Show the data in a simple table. This is the only option if the Node graph panel is unavailable.
- 
Or something like that

Reply:
Yes, but add a link to the Explore topic in the bullet.